build.gradle

```diff
@@ -20,5 +21,20 @@ buildscript {
 allprojects {
     repositories {
         jcenter()
+        maven { url "https://www.jitpack.io" }
     }
 }
+
+ext {
+    // Sdk and tools
+    minSdkVersion = 11
+    targetSdkVersion = 25
```

Comment:
This 'targetSdkVersion' is the range up to which our app will work. So it is not necessary to have Android 7.1 device for testing.